README.md

```diff
@@ -10,6 +10,12 @@
 
 solidus_cmd is a command line utility for creating extensions for the [Solidus ecommerce platform](https://github.com/solidusio/solidus).
 
+## DEPRECATION NOTICE :warning: :construction:
+
+This extension is deprecated in favor of [Solidus extension dev tools](https://github.com/solidusio-contrib/solidus_extension_dev_tools).
+
+Use it to generate new Solidus extension.
```

Comment:
It's not clear what "it" refers to here. I think we can remove this sentence completely actually.